src/main/controllers/AttachmentController.ts

```diff
@@ -26,22 +26,20 @@ export default class AttachmentController {
     let supportingFileId;
 
     if (req.session.documentDownloadPage === PageUrls.RESPONDENT_APPLICATION_DETAILS) {
-      respondentAppDocId = getDocId(userCase.selectedGenericTseApplication?.value.documentUpload.document_url);
-      const claimantResponseDoc = getClaimantResponseDocDownload(userCase.selectedGenericTseApplication);
-      claimantResponseDocId = getDocId(claimantResponseDoc.document_url);
-      decisionDocId = userCase.selectedGenericTseApplication?.value.adminDecision
-        .map(it => getDocId(it.value.responseRequiredDoc.document_url))
-        .find(id => id === docId);
+      respondentAppDocId = getDocId(userCase.selectedGenericTseApplication?.value.documentUpload?.document_url);
+      claimantResponseDocId = getClaimantResponseDocId(userCase.selectedGenericTseApplication);
+      decisionDocId = getDecisionDocId(req, userCase.selectedGenericTseApplication);
```

Comment:
This is a long chain, could it be broken down using a variable? 
userCase.selectedGenericTseApplication is used 3 times, can this be put into a variable?

Reply:
Put in variable